tests/xtimer_cli/README.md

```diff
@@ -0,0 +1,14 @@
+# Xtimer API test
+
+This application enables tests of xtimer API calls using the RIOT shell.
+Consult the 'help' shell command for available actions.
+
+## Requirements
+
+The automated tests are based on RobotFramework and thus depend on additional
+Python packages, namely `robotframework` and `riot_pal`.
```

Comment:
maybe add a reference to dist/robotframework/README.md for installing the requirements here?

Reply:
yep makes sense